openvpn/nordvpn/configure-openvpn.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
For the container I don’t see any point changing this

Reply:
I've read somewhere (can't find the source though), that a logical she-bang (env ...) was preferred over an absolute one. this link sums up both she-bang types: https://unix.stackexchange.com/questions/29608/why-is-it-better-to-use-usr-bin-env-name-instead-of-path-to-name-as-my
So now by default, I put a logical she-bang in my scripts, for the container, you are right there is no advantage to have this change as path to bash will not change. If this change is blocking the PR, I will push a revert commit to restore the line